README.md

```diff
@@ -34,6 +34,7 @@ See https://jekyllrb.com/docs/plugins/installation/ .
 ## Usage
 
 1. Create `_cache` directory.
+   * This directory _must_ exist under your project root even if you've modified the [site source](https://jekyllrb.com/docs/configuration/options/).
```

Comment:
Could you also add a note in `Custom templates` section that `_includes` directory must be under `source` directory?

Reply:
Fixed as requested.